### DISCUSSION TITLE
🔥 Tech Forum 기술 글자랑 대회 2024 🔥

### DISCUSSION THREAD

**Discussion by anjiniii** [DeveloperAcademy-POSTECH/.github#79] [Announcements]
## 🎉 Tech Forum 기술 글자랑 대회 🎉

안녕하세요, Tech Forum 메인테이너입니다. 🍀

**10월 21일 월요일부터 11월 17일 일요일**까지 4주간 Tech Forum에서 **기술 글자랑 대회**가 열립니다
매주 새로운 주제가 공개되며, 제시된 주제 또는 자유 주제로 **Academy Github Discussion 채널**에 글을 작성해 주세요.
이 기간 동안 작성된 모든 글은 자동 대회 참여로 확인합니다.

이번 대회는 Macro Challenge 팀 또는 개인으로 참여할 수 있습니다.
4주 동안 성실하게 참여한 팀과 개인에게는 시상과 함께 특별한 혜택이 주어질 예정이니 많은 관심과 참여 부탁드립니다!

글 작성에 앞서, 글의 첫 부분에 `#기 ##입니다.`(예: 3기 이오입니다.) 를 작성해 작성자를 꼭 알려주세요!

### 행사 규칙
- CoC 행동 규범을 꼭 읽고 준수해 주세요.
- 글은 최소 300자 이상 작성해 주세요.
- 단순히 링크만 공유된 글은 심사 대상에서 제외됩니다.
- 기술/개발과 관련된 내용이라면 어떤 주제든 가능합니다.

### 글자랑 주제
- 1주차(10/21 - 10/27): 개발 협업 🙌
- 2주차(10/28 - 11/3): 트러블 슈팅 🔥
- 3주차(11/4 - 11/10): 내가(팀이) 경험한 기술적인 도전 🧪
- 4주차(11/11 - 11/17): 더 나은 코드, 리팩토링 🛠️

### 문의 사항
궁금한 사항은 이 글의 댓글로 남겨주시거나,
Tech Forum 메인테이너 (오전: 라뮤 🎨, 모수 🦒, 빈치 🐨, 헤일 🌊, 제인 🍭 / 오후: 이오🍦, 구리스 🐟 / 종일: 예니 👻)에게 문의해 주세요.
러너 여러분의 많은 관심과 참여 부탁드립니다! 🍀

<br>
<br>

![image](https://github.com/user-attachments/assets/85c5c04c-3538-4dbf-8ceb-c9ba3949b4f4)

**Comment by yeniful** [comment]
우와!!!! 드디어 시작하는 글자랑 대회!!!!! (멘토도 참여해도 되나요!!!)
